src/Microsoft.FeatureManagement.AspNetCore/FeatureGateAttribute.cs

```diff
@@ -133,7 +133,7 @@ public async Task OnPageHandlerExecutionAsync(PageHandlerExecutingContext contex
             IFeatureManagerSnapshot fm = context.HttpContext.RequestServices.GetRequiredService<IFeatureManagerSnapshot>();
 
             //
-            // Enabled state is determined by either 'any' or 'all' features being enabled.
+            // FeatureFlagEnabled state is determined by either 'any' or 'all' features being enabled.
```

Comment:
I don't understand this change

Reply:
Sorry, my bad. This is a mistake caused by rename. I click the "Include comments" and "Include strings" by mistake.